src/components/ApplicationsModal/ApplicationsModal.tsx

```diff
@@ -305,8 +305,13 @@ export const ApplicationsModal: React.FC<ApplicationsModalProps> = ({
 
   /* App-related actions */
 
-  const checkForLabels = (stringOfLabels: string) => {
-    return stringOfLabels.length ? stringOfLabels.split(" ") : [];
+  const checkForLabels = (labels: string) => {
+    return !labels.length
```

Comment:
If we're going to filter no need to check if the `labels` have length.

```js
  return labels.split(",")
    .map((l) => l.trim())
    .filter((l) => l.length);
```

Reply:
ok